sass/_components.scss

```diff
@@ -2458,6 +2474,64 @@
             font-size: $size-3;
         }
     }
+
+    ul {
+        margin-left: 0;
+        list-style: none;
+
+        li {
+            position: relative;
+            &:before {
+                margin-top: 0;
+                left: -$baseline;
+                margin-left: 0;
+                position: absolute;
+                content: '';
+                top: -5px;
+                border-top: 1rem solid transparent;
+                border-bottom: 1rem solid transparent;
```

Comment:
bit weird - why not padding/margin/top etc?

Reply:
These border styles are the way I create the arrows. It seems a bit hacky, I agree, but it works well